Plugins/Flow.Launcher.Plugin.ProcessKiller/PortHelper.cs

```diff
@@ -0,0 +1,179 @@
+        {
+            return string.Format(@" Process Name: {0} ,Process ID: {1} ,
+                                    Port: {2} ,\nPath : {3}", ProcessName,
+                                    ProcessID, Port, Path);
+        }
+
+    }
+    /// <summary>
+    /// Usage:
+    /// int port = 8081
+    /// TcpHelperUtil tcpHelper = new TcpHelperUtil();
+    /// var details = tcpHelper.GetPortDetails(port);
+    /// if (details.Item1)
+    /// {
+    ///     Console.WriteLine("Port {0} in Use",port);
+    ///     Console.WriteLine(details.Item2.ToString());
+    /// }else
+    /// {
+    ///     Console.WriteLine("Port {0} is free ",port);
+    /// }
+    ///
+    /// </summary>
+    internal class PortHelper
+    {
+        private const short MINIMUM_TOKEN_IN_A_LINE = 5;
+        private const string COMMAND_EXE = "cmd";
+
+        public PortHelper()
+        {
+
+        }
+
+        public static Tuple<bool, PortDetail> GetPortDetails(int port)
+        {
+            PortDetail PortDetail = new PortDetail();
+            Tuple<bool, PortDetail> result = Tuple.Create(false, PortDetail);
+
+            // execute netstat command for the given port
+            string commandArgument = string.Format("/c netstat -an -o -p tcp|findstr \":{0}.*LISTENING\"", port);
+
+            string commandOut = ExecuteCommandAndCaptureOutput(COMMAND_EXE, commandArgument);
+            if (string.IsNullOrEmpty(commandOut))
+            {
+                // port is not in use
+                return result;
+            }
+
+            var stringTokens = commandOut.Split(default(Char[]), StringSplitOptions.RemoveEmptyEntries);
+            if (stringTokens.Length < MINIMUM_TOKEN_IN_A_LINE)
+            {
+                return result;
+            }
+
+            // split host:port
+            var hostPortTokens = stringTokens[1].Split(new char[] { ':' });
+            if (hostPortTokens.Length < 2)
+            {
+                return result;
+            }
+
+            int portFromHostPortToken = 0;
+            if (!int.TryParse(hostPortTokens[1], out portFromHostPortToken))
+            {
+                return result;
+            }
+            if (portFromHostPortToken != port)
+            {
+                return result;
+            }
+
+            PortDetail.Port = port;
+            PortDetail.ProcessID = int.Parse(stringTokens[4].Trim());
+            Tuple<string, string> processNameAndPath = null;
+            try
+            {
+                processNameAndPath = GetProcessNameAndCommandLineArgs(PortDetail.ProcessID);
+                PortDetail.ProcessName = processNameAndPath.Item1;
+                PortDetail.Path = processNameAndPath.Item2;
+                PortDetail.Process = Process.GetProcessById(PortDetail.ProcessID);
+                result = Tuple.Create(true, PortDetail);
+            }
+            catch (Exception exp)
+            {
+                Console.WriteLine(exp.ToString());
```

Comment:
Don't use `Console.WriteLine`. Try to use Log in the `IPublicAPI` inside `PluginInitContext`.